effekt/shared/src/main/scala/effekt/generator/js/Transformer.scala

```diff
@@ -134,6 +134,17 @@ trait Transformer {
       }
     }.toString()
 
+  // Externs
+  // -------
+  def normalizeExternStrings(l: List[String])(using Context): List[String] = {
+    val start = l.head
+    val end = l.last
+    val mid = l.tail.init
+    if (start.matches("""^\s.*""") || end.matches(""".*\s$""")) {
+      Context.warning("Extern string in js has trailing / leading whitespace. This will be removed.")
```

Comment:
I think this warning can be very annoying, no?

Reply:
It could be - we're not using it anywhere yet, so...
I can also remove it.